background.ts

```diff
@@ -150,7 +150,11 @@ Electron.app.whenReady().then(async() => {
     }
 
     installDevtools();
-    setupProtocolHandler();
+    if (process.env.NODE_ENV && ['development', 'test'].includes(process.env.NODE_ENV)) {
```

Comment:
```suggestion
    if (/^(?:dev|test)/i.test(process.env.NODE_ENV ?? '')) {
```
(To match the other occurrence)

Reply:
Makes sense to me